### PR TITLE
 add some attributes to the Options of a Connection

### PR DESCRIPTION
**Pull request by laurentdutheil**
- enable_database_customization
- brut_force_protection
- import_mode
- disable_signup

### CODE REVIEW

**Comment by laurentdutheil** [comment]
Hello

This PR depends of https://github.com/yieldr/go-auth0/pull/4

I don't know how to reference the new dependency (I'm a newbe in go).
Could you explain it to me ?

**Comment by alexkappa** [comment]
I believe you need to run `dep ensure -update github.com/yieldr/go-auth0` to update the go-auth0 dependency.

File: auth0/resource_auth0_connection.go [hunk 1]
```diff
@@ -107,6 +107,22 @@ func newConnection() *schema.Resource {
 							Type:     schema.TypeBool,
 							Optional: true,
 						},
+						"enable_database_customization": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"brut_force_protection": {
```
[the remark this answers is not in the document]
Comment (laurentdutheil) [reply auth0/resource_auth0_connection.go]: This is mine :(
I fix it in the go-auth0 project and this one

**Comment by laurentdutheil** [comment]
I tried to execute `dep ensure -update github.com/yieldr/go-auth0`

But I think it updates to the version 0.1.1, not the last commit of the repo.
Did I miss something ?

**Comment by laurentdutheil** [comment]
I think I find a workaround.
I edit the Gopkg.lock with the right commit of the yieldr/go-auth project.
Then I execute `dep ensure  -vendor-only`

**Pull request merged**